<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>proyecto_1</class>
 <widget class="QMainWindow" name="proyecto_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>proyecto_1</string>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="inicioBoton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>INICIO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="carnetLine">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelCarnet">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N° Carnet:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="velLCD">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelParametros">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Parámetros medidos del motor</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelVel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Velocidad:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="frecLCD">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>172</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelFrec">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Frecuencia:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="volLCD">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelVol">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>65</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltaje:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="tempLCD">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelTemp">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperatura:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRpm">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>R.p.m </string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHz">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>190</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hz    </string>
    </property>
   </widget>
   <widget class="QLabel" name="labelV">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>240</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>V     </string>
    </property>
   </widget>
   <widget class="QLabel" name="labelC">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>290</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>°C    </string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="segLCD">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>340</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>T. operación:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="tolVel">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>120</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>7</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="tolFrec">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>170</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>7</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="tolVol">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>220</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>7</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="tolTemp">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>270</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>7</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="tVel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>122</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>7</number>
    </property>
    <property name="value" stdset="0">
     <double>1800.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="labelTeoricos">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>90</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Valores teóricos</string>
    </property>
   </widget>
   <widget class="QLabel" name="labeTol">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>90</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tolerancia</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="tfrec">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>170</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>7</number>
    </property>
    <property name="value" stdset="0">
     <double>60.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="tvol">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>220</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>7</number>
    </property>
    <property name="value" stdset="0">
     <double>120.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="ttemp">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>270</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>7</number>
    </property>
    <property name="value" stdset="0">
     <double>40.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="labelRpm1">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>130</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>R.p.m ±</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHz1">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>180</y>
      <width>65</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Hz    ±</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelV1">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>230</y>
      <width>65</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>V     ±</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelC1">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>280</y>
      <width>65</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>°C    ±</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelpor1">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>130</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelpor2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>180</y>
      <width>65</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelpor3">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>230</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelpor4">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>280</y>
      <width>65</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="minLCD">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>340</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="horasLCD">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>64</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="diasLCD">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelPuntos1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>340</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPuntos2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>340</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dia">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>390</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DIAS</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHora">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>390</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HORAS</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMinuto">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>390</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MINUTOS</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSeg">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>390</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SEGUNDOS</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>827</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
